controller/app.controller.js

```diff
@@ -22,4 +22,24 @@ exports.getAllEndpoints = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params
+
+    if (!Number(article_id)){
+        return res.status(400).send({message: "invalid id path"})
+    }
+
+    fetchArticleById(article_id)
+    .then((response) => {
+        res.status(200).send({response})
+    })
+    .catch((err) => {
+        if (err) {
+            res.status(404).send({message: "article not found"})
+        } else{
```

Comment:
If we make it into this block the err does not exist, so we can't invoke next with it